Open Door System with key

0shiros opened the pull request:
Player can open Door if he has the key (key used after open door) (Prefab in Rooms)

0shiros committed:
Open Door System with key

Player can open Door if he has the key (key used after open door) (Prefab in Rooms)

DoriannX commented on Assets/Scripts/Entities/PlayerScripts/Player.cs:
```diff
@@ -103,4 +109,26 @@ private void UpdateUI()
         ui.Update(this);
 }
 */
+    public void Interact()
```
met ça dans un script player interact

0shiros committed:
Create PlayerInteract script

Move method Interact from Player to new script PlayerInteract + Modify inputSystemAction to interact with F instead of E

0shiros committed:
Modify script reference

Input call player which call playerInteract

DoriannX committed:
fixing merge

DoriannX committed:
Merge branch 'Dev' into Feat_DoorSystem
